src/v1/util/handlers.js

```diff
@@ -12,7 +12,7 @@ export function validationError(err, req, res, next) {
   return next(err);
 }
 
-export function internalError(err, req, res, next) {
+export function internalError(err, req, res) {
```

Comment:
Os errors handlers do express tem que receber 4 argumentos, mesmo se não for utilizar o next ele tem que ficar ali =/
http://expressjs.com/en/guide/error-handling.html